.ci/acceptance-tests/ansible-integration.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# CI sets the contents of our json account secret in our environment; dump it
+# to disk for use in tests.
+echo "${SERVICE_ACCOUNT_KEY}" > /tmp/google-account.json
+echo "${ANSIBLE_TEMPLATE}" > /tmp/ansible-template.yml
+
+set -e
+set -x
+
+pushd magic-modules-new-prs/build/ansible
+
+# Setup Cloud configuration template with variables
+cp /tmp/ansible-template.yml test/integration/cloud-config-gcp.yml
+
+# Install dependencies for ansible
```

Comment:
This belongs in container setup rather than in the tests themselves.  You should include `pip install -r requirements.txt` in this file, just in case the requirements change, but you can speed things up by pre-installing.

Reply:
Agreed!